test/integration/provision/os/rancheros.bats

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bats
+
+load ${BASE_TEST_DIR}/helpers.bash
+
+force_env DRIVER virtualbox
+
+export RANCHEROS_VERSION="v0.3.1"
+export RANCHEROS_ISO="https://github.com/rancherio/os/releases/download/$RANCHEROS_VERSION/machine-rancheros.iso"
+
+@test "$DRIVER: create with RancherOS ISO" {
+  VIRTUALBOX_BOOT2DOCKER_URL="$RANCHEROS_ISO" run ${BASE_TEST_DIR}/run-bats.sh ${BASE_TEST_DIR}/core
```

Comment:
This test actually doesn't work with _all_ of the tests in `core` due to a variety of issues (e.g. engine options test expects `--engine-storage-driver devicemapper` to be allowed with RancherOS, which it isn't), so I will probably change it around a bit, but I wanted to get it out the door as a POC